### PR TITLE
Docker wrapper + Course, Ball and Tournament Updates

### PR DESCRIPTION
**Pull request by Yardboy**
I added a Docker help doc describing how to run things, but that would require you to install Docker and know a little something about it. All of this lays on top of your original codebase, so rather than testing the Docker stuff out yourself, I think what's more important is that you make sure that *your* dev environment + ability to generate the site hasn't been affected by the Docker stuff I added. Let me know if you run into any issues.

I've also now added new balls up to Athlete, and new tournaments City of Light and Silver State. No Celtic Cup yet because I don't have images for the new course.

Added Drumore Links course and Celtic Cup tournament after I realized Vic had previously sent me images. I'll update later with resources for the individual holes - done for the night.

Should the Silver State title banner be resized differently? It's causing a push on the grid layout.

### CODE REVIEW

File: modules/site/src/main/resources/microsite/_config.yml [hunk 1]
```diff
@@ -750,3 +757,27 @@ page_gen:
...
+      title-prefix: 'City of Light - Hole '
+      tournamentName: 'City Of Light'
+  - index_files: true
+    data: 'tournaments.SilverState.holes'
+    template: '_includes/tournaments/hole-template.liquid'
+    name: 'number'
+    dir: 'tournaments/SilverState'
+    extra:
+      title-prefix: 'Silver State - Hole '
+      tournamentName: 'Silver State'
```
Comment (cranst0n) [inline modules/site/src/main/resources/microsite/_config.yml]: Monster Marsh entry needed here.

File: modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+courseName: Drumore Links
+skinned: false
+holes:
+  - id: e9d88d48-361e-41d2-bd48-0bf29109c130
+    course: Drumore Links
+    number: 1
+    par: 4
+    replayName: "1"
```
Comment (cranst0n) [inline modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml]: Do we have the real replay names available for these holes? Not a blocker but would be nice.

File: modules/site/src/main/resources/microsite/data/tournaments/MonsterMash.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+tournamentName: Monster Mash
```
Comment (cranst0n) [inline modules/site/src/main/resources/microsite/data/tournaments/MonsterMash.yaml]: Typo: Should be Monster ***Marsh***

**Comment by cranst0n** [comment]
And yes the Silver State image needs to be resized to 550x200.

File: modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml (hunk 2 above)
Comment (Yardboy) [reply modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml]: @cranst0n I do not, unfortunately no one paid attention during the tournament and I'm not sure how I get them now.

File: modules/site/src/main/resources/microsite/_config.yml (hunk 1 above)
Comment (Yardboy) [reply modules/site/src/main/resources/microsite/_config.yml]: Update coming shortly will fix this.

**Comment by Yardboy** [comment]
I've pushed up the requested changes.

**Comment by Yardboy** [comment]
I've also updated the Docker wrapper to use Bundler 1.17.2 which should match your env.

File: modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml (hunk 2 above)
Comment (cranst0n) [reply modules/site/src/main/resources/microsite/data/courses/DrumoreLinks.yaml]: You can find them in the in-game Replay section after you've played the hole. It's a shame PD doesn't make this info easier to find/share but their track record for making meaningful improvements is what it is....poor. But as I said, this isn't a blocker and can be added at a later date.

**Pull request merged**